### PR TITLE
TASK: Streamline styling of clear buttons in node tree

### PR DESCRIPTION
**Pull request by aertmann**

### CODE REVIEW

**Comment by aertmann** [comment]
image
image
size, font and hover changed

**Comment by aertmann** [comment]
:+1:

File: TYPO3.Neos/Resources/Public/Styles/Neos.css [hunk 1]
```diff
@@ -8050,27 +8050,23 @@
   background-image: none !important;
 }
 .neos #neos-context-bar .chosen-container .chosen-single abbr {
-  background: #fff !important;
-  top: 11px;
+  top: 12px;
   right: 48px;
```
Comment (gerhard-boden) [inline TYPO3.Neos/Resources/Public/Styles/Neos.css]: `right: 54px` would give the button the same offset to the right as the magnifier in the search field.(you kinda notice it when seeing them side by side).
 The close button (as well as the magnifier) are not vertical centered, but that does not stand out much.

**Comment by gerhard-boden** [comment]
tiny nitpick, otherwise looks good and works great in:
FF 42 (Windows)
Chrome 47.0.2526.106 (Windows)
Internet Explorer 11.0.26 (Windows)

**Comment by aertmann** [comment]
agreed, thanks for the reviewing and the suggestion

**Pull request merged**